ckine/figures/figureCommon.py

```diff
@@ -282,12 +282,20 @@ def calc_dose_response(cell_names, unkVec, receptor_data, tps, cytokC, expr_act2
     total_activity15 = total_activity2.copy()
 
     for i, cell in enumerate(cell_names):
-        # updates rxntfr for receptor expression for IL2Ra, IL2Rb, gc
-        cell_data = receptor_data[i]
-        rxntfr2[:, 22] = receptor_expression(cell_data[0], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
-        rxntfr2[:, 23] = receptor_expression(cell_data[1], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
-        rxntfr2[:, 24] = receptor_expression(cell_data[2], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
-        rxntfr2[:, 25] = 0.0  # We never observed any IL-15Ra
+        if Traf:
+            # updates rxntfr for receptor expression for IL2Ra, IL2Rb, gc
+            cell_data = receptor_data[i]
+            rxntfr2[:, 22] = receptor_expression(cell_data[0], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
+            rxntfr2[:, 23] = receptor_expression(cell_data[1], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
+            rxntfr2[:, 24] = receptor_expression(cell_data[2], rxntfr2[:, 17], rxntfr2[:, 20], rxntfr2[:, 19], rxntfr2[:, 21])
+            rxntfr2[:, 25] = 0.0  # We never observed any IL-15Ra
+        else:
+            # updates rxntfr for receptor expression for IL2Ra, IL2Rb, gc
+            cell_data = receptor_data[i]
+            rxntfr2[:, 22] = cell_data[0] / 100
```

Comment:
You can scale the pSTAT5 signal proportionally, since that is measured using arbitrary fluorescence units. You can't arbitrarily scale the receptor amounts though. What exactly was the problem when these weren't scaled?

Reply:
No problem - will fix. The issue is that the units are magnitudes higher and the confidence interval appears extremely thin. I thought that the units jump might confuse people who aren't savvy to their arbitrary status, but it was a minor point. Will restore this now.

Reply:
You could scale the results from the model calculation?

Reply:
Ok just did this. Something still feels weird to me, as the output is *so* far off from the trafficking model. Are you sure we can input the raw receptor quantities into the no trafficking model? The no_traf fits have rexpr values that range from 0-10 or so, so inputting things on the order of 10^4 in these unknowns gives me pause. Of course none of this means that it's wrong, I'm just surprised it looks this poor.

Reply:
The code handles these completely differently for the no-trafficking model, so it doesn't mean anything to compare their scales.

Actually, I think this is right, and I noticed a related error in Fig. S1. Fig. S1C should have units of #/cell, not #/cell/min. While the no-trafficking model fits the YT-1 data, it does so by inferring a minuscule amount of receptor. By contrast, the full trafficking model infers that the YT-1 cells have comparable amounts of receptor to PBMCs. So, when you plug in the PBMC data, the trafficking model does well, and the no-trafficking model is thrown way off.

So, we can merge this, fix that axis in S1C, and then add a couple notes in the text about this! In the abstract, we can adjust to say that although a trafficking model fit about as well, it was absolutely required to extrapolate to these new predictions.

Reply:
Ok perfect, I made those changes and will discuss these findings in the text.